cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,24 @@
-/// <reference types="cypress" />
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
+    cy.visit('https://conduit.mate.academy/user/login');
+    cy.url().should('include', '/login');
+  });
+
+  it('should successfully submit the email and password fields', () => {
```

Comment:
remove this test because you do the same actions in the next test. This is code duplication

Reply:
Removed